apps/omg/test/omg/state/core_test.exs

```diff
@@ -615,9 +615,7 @@ defmodule OMG.State.CoreTest do
              }, _, _}, _} = form_block_check(state)
 
     # precomputed fixed hash to check compliance with hashing algo
-    assert block_hash ==
-             <<168, 134, 99, 72, 67, 39, 246, 127, 88, 163, 200, 169, 167, 105, 15, 124, 143, 240, 174, 27, 84, 112, 48,
-               53, 41, 53, 236, 243, 192, 188, 116, 226>>
+    assert <<220, 51, 45, 150, 11, 157, 177, 120, 76, 168>> <> _ = block_hash
```

Comment:
Does it makes any difference checking the first 10 bytes instead of full value? It does not. 
I'm against checking any kind of such values so when we have a test where value of the hash is important for compliance I'm opting to compare on full value.

Reply:
so this is not compliance (conformance) really, just "invariance", so a test that goes red whenever you touch hashing, so that you know that you touched hashing. This convention is followed in other tests like this (albeit some consistency more would help - separate cleanup PRs pending).

I just thought when changing this line anyway, I'll adapt to the (I think) cleanest convention out there. WDYT?